optimum/intel/openvino/configuration.py

```diff
@@ -44,14 +44,13 @@ class OVConfig(BaseConfig):
     FULL_CONFIGURATION_FILE = "ov_config.json"
 
     def __init__(
-        self,
-        compression: Optional[Dict] = None,
-        input_info: Optional[List] = None,
-        **kwargs,
+        self, compression: Optional[Dict] = None, input_info: Optional[List] = None, dump_onnx: bool = False, **kwargs
     ):
         super().__init__()
         self.compression = compression or DEFAULT_QUANTIZATION_CONFIG
         self.input_info = input_info
+        self.dump_onnx = dump_onnx
```

Comment:
I would rename it to 
```suggestion
        self.save_onnx_model = save_onnx_model
```